2025-04-03 14:30:00,380 - INFO - Calling fetch_google_sheets_docs.py...
2025-04-03 14:30:00,472 - ERROR - Error:
python: can't open file 'C:\\Users\\Checkout\\cron-job-service\\fetch_google_sheets_docs.py': [Errno 2] No such file or directory

2025-04-03 14:30:00,474 - INFO - fetch_google_sheets_docs.py executed successfully.
2025-04-03 14:30:00,475 - INFO - Sleeping for 300 seconds...

2025-04-03 14:33:17,009 - INFO - Calling fetch_google_sheets_docs.py...
2025-04-03 14:33:17,318 - ERROR - Error:
Traceback (most recent call last):
  File "C:\Users\Checkout\cron-job-service\scripts\fetch_google_sheets_docs.py", line 7, in <module>
    from googleapiclient.discovery import build
ModuleNotFoundError: No module named 'googleapiclient'

2025-04-03 14:33:17,320 - INFO - fetch_google_sheets_docs.py executed successfully.
2025-04-03 14:33:17,321 - INFO - Sleeping for 300 seconds...

2025-04-03 15:00:35,715 - Calling fetch_google_sheets_docs.py...
2025-04-03 15:00:36,413 - fetch_google_sheets_docs.py executed successfully.
2025-04-03 15:00:36,413 - Sleeping for 300 seconds...

2025-04-03 15:02:07,755 - Calling fetch_google_sheets_docs.py...
2025-04-03 15:02:07,810 - Error output of fetch_google_sheets_docs.py:
python: can't open file 'C:\\Users\\Checkout\\cron-job-service\\fetch_google_sheets_docs.py': [Errno 2] No such file or directory

2025-04-03 15:02:07,810 - fetch_google_sheets_docs.py executed successfully.
2025-04-03 15:02:07,810 - Sleeping for 300 seconds...

